pidtree_bcc/main.py

```diff
@@ -87,9 +104,12 @@ def termination_handler(probe_workers: List[Process], signum: int, frame: Any):
     :param int signum: signal integer code
     :param Any frame: signal stack frame
     """
-    logging.warning('Caught termination signal, shutting off probes and exiting')
+    msg_info = ('restart', 'restarting') if signum == signal.SIGHUP else ('termination', 'exiting')
+    logging.warning('Caught {} signal, shutting off probes and {}'.format(*msg_info))
     for worker in probe_workers:
         worker.terminate()
+    if signum == signal.SIGHUP:
```

Comment:
What's this going to do if pidtree's running in an interactive session and the user hangs up?

It might make more sense to use SIGUSR1 if this is going to cause a problem.

Reply:
I guess you could say the same of many other pieces of software, e.g. uwsgi, so I don't really see it as a problem.